ontology/yaml/resources/HVAC/entity_types/WSHP.yaml

```diff
@@ -27,3 +27,20 @@ WSHP_SS_VSFC_HWDT_POTHWSC:
   - HWDT
   - POTHWSC
 
+WSHP_FDPM_CWDT_SFVSC_CDCPC_CAM_ZTC_RTM_CWDT_CDWISOVM_STM:
+  guid: "e1b4a911-00a8-42da-abaf-8eff6987ad58"
+  description: "Cooling only water source heat pump using condensor water to carry heat away from the zone.  One filter, condensor water pump control, one compressor, condensate alarm"
+  is_canonical: true
+  implements:
+  - WSHP
+  - FDPM
+  - CWDT
```

Comment:
You use `CWDT` twice, please fix.

Reply:
I sent an invite for a discussion tomorrow.  This WSHP has return air drawn through a coil.  In the coil is condenser water from the plant.  The condenser water is cooled via DX with a compressor using refrigerant.  Heat is taken out of the space and put into the condenser water loop.